_data/iot.yml

```diff
@@ -93,9 +93,10 @@ websites:
     img: totalconnect.png
 
   - name: Wyze
-    url: https://www.wyzecam.com
+    url: https://www.wyze.com
     img: wyze.png
     tfa:
       - sms
+      - totp
     doc: https://support.wyzecam.com/hc/en-us/articles/360024402052
```

Comment:
Note that this link has not changed to `wyze.com`, though. It actually redirects to https://wyzelabs.zendesk.com/hc/en-us/articles/360024402052, but in case Wyze changes helpdesk vendors someday, the current URL seems closer to being a permalink.